replaces hardcoded paths to xlib and opengl by EB paths in qmake.conf
author: Alex Domingo, originally written for Qt-5.12.3, see pr #8544
diff -Nru qtbase/mkspecs/linux-g++-64/qmake.conf qtbase/mkspecs/linux-g++-64/qmake.conf
--- qtbase/mkspecs/linux-g++-64/qmake.conf	2019-04-09 11:51:26.000000000 +0200
+++ qtbase/mkspecs/linux-g++-64/qmake.conf	2019-06-04 00:26:20.921468000 +0200
@@ -18,7 +18,7 @@
 include(../common/g++-unix.conf)
 
 
-QMAKE_LIBDIR_X11        = /usr/X11R6/lib64
-QMAKE_LIBDIR_OPENGL     = /usr/X11R6/lib64
+QMAKE_LIBDIR_X11        = $$(EBROOTX11)/lib
+QMAKE_LIBDIR_OPENGL     = $$(EBROOTLIBGLU)/lib
 
 load(qt_config)
diff -Nru qtbase/mkspecs/linux-icc-64/qmake.conf qtbase/mkspecs/linux-icc-64/qmake.conf
--- qtbase/mkspecs/linux-icc-64/qmake.conf	2019-04-09 11:51:26.000000000 +0200
+++ qtbase/mkspecs/linux-icc-64/qmake.conf	2019-06-04 00:28:29.070834000 +0200
@@ -12,5 +12,5 @@
 
 # Change the all LIBDIR variables to use lib64 instead of lib
 
-QMAKE_LIBDIR_X11        = /usr/X11R6/lib64
-QMAKE_LIBDIR_OPENGL     = /usr/X11R6/lib64
+QMAKE_LIBDIR_X11        = $$(EBROOTX11)/lib
+QMAKE_LIBDIR_OPENGL     = $$(EBROOTLIBGLU)/lib
diff -Nru qtbase/mkspecs/openbsd-g++/qmake.conf qtbase/mkspecs/openbsd-g++/qmake.conf
--- qtbase/mkspecs/openbsd-g++/qmake.conf	2019-04-09 11:51:26.000000000 +0200
+++ qtbase/mkspecs/openbsd-g++/qmake.conf	2019-06-04 00:31:07.877995000 +0200
@@ -12,10 +12,10 @@
 QMAKE_LIBDIR_POST       = /usr/local/lib
 
 # System provided X11 defaults to X11R6 path on OpenBSD
-QMAKE_INCDIR_X11        = /usr/X11R6/include
-QMAKE_LIBDIR_X11        = /usr/X11R6/lib
-QMAKE_INCDIR_OPENGL     = /usr/X11R6/include
-QMAKE_LIBDIR_OPENGL     = /usr/X11R6/lib
+QMAKE_INCDIR_X11        = $$(EBROOTX11)/include
+QMAKE_LIBDIR_X11        = $$(EBROOTX11)/lib
+QMAKE_INCDIR_OPENGL     = $$(EBROOTLIBGLU)/include
+QMAKE_LIBDIR_OPENGL     = $$(EBROOTLIBGLU)/lib
 
 QMAKE_RPATHDIR         += $$QMAKE_LIBDIR_X11
 
